src/main/java/club/sk1er/patcher/mixins/features/GuiNewChatMixin_TransparentChat.java

```diff
@@ -5,12 +5,21 @@
 import net.minecraft.client.gui.Gui;
 import net.minecraft.client.gui.GuiNewChat;
 import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.Shadow;
 import org.spongepowered.asm.mixin.injection.At;
 
 @Mixin(GuiNewChat.class)
 public abstract class GuiNewChatMixin_TransparentChat extends Gui {
+
+    @Shadow
+    public abstract boolean getChatOpen();
+
+    @SuppressWarnings("unused")
```

Comment:
Not needed, the editor just doesn't recognize mixinextra annotations as mixin annotations yet.